### ISSUE TITLE
MY computer just freezes and then restarts by itself when i use this to train on my dataset. 

### ISSUE THREAD

**Issue by manoja328** [ruotianluo/pytorch-faster-rcnn#58]
MY computer just freezes and then restarts by itself when i use this to train on my dataset. Can  anyone guess how this could occur. I compilde all the files as said in the README  using (sm_52 for TItanX) GPU ?

**Comment by ruotianluo** [comment]
Does it work on coco?

**Comment by manoja328** [comment]
I tried it on pascal VOC and  it ran fine.

**Comment by ruotianluo** [comment]
It may because of your own dataset. I think the most reliable way to debug is first locate where the program crashes.

**Comment by manoja328** [comment]
yeah that would be good if i could see anything on the screen . Everything just freezes even the monitor go dark and i have to restart the machine.  Could bad inputs to the gpu i.e bad co-rdinates while performing NMS or other things in GPU can cause this?

**Comment by ruotianluo** [comment]
I'm not sure about that. Try just write progresses into a log file and flush it every time after each write. This is the only way I can think of that can locate the crash. I don't know if there is debug flag to run python that can automatically generate some log file.

**Comment by manoja328** [comment]
Thanks . Will try that.

**Comment by EldorIbragimov** [comment]
I think it is because of memory. I have 2000 image dataset. when I trained it. I just check the memory usage it was more than 200 gb.
